lldb/source/Plugins/ScriptInterpreter/Python/ScriptInterpreterPython.cpp

```diff
@@ -2755,6 +2755,58 @@ bool ScriptInterpreterPythonImpl::RunScriptBasedCommand(
   return ret_val;
 }
 
+bool ScriptInterpreterPythonImpl::RunScriptBasedParsedCommand(
+    StructuredData::GenericSP impl_obj_sp, Args &args,
+    ScriptedCommandSynchronicity synchronicity,
+    lldb_private::CommandReturnObject &cmd_retobj, Status &error,
```

Comment:
Why return a bool instead of a `Status` object? You've got an error mutable ref already, that could just be the return value

Reply:
In this patch, I was just adding the Parsed command to the Raw one that was already there, without changing more than I needed to.  If we want to go back and clean up little things like this in both interfaces, I'd rather do that as a separate piece of work.

Reply:
Makes sense to me.